docs/api/desktop-capturer.md

```diff
@@ -5,8 +5,7 @@
 
 Process: [Main](../glossary.md#main-process)
 
-The following example shows how to capture video from a desktop window whose
-title is `Electron`:
+The following example shows how to capture video and system audio (if supported) of the current desktop:
```

Comment:
I don't understand what the "title is Electron" part has to do with the example below. Let's change it to "current desktop" since that's more apt.